### ISSUE TITLE
Issues tracker instead of dashboard

### ISSUE THREAD

**Issue by nmanovic** [openvinotoolkit/cvat#351]
It is better to have a very powerful issues tracker instead of our default dashboard to track annotation tasks, assign them on different users, build statistics and visualizations. Need to integrate CVAT with an external tracker or implement an own simple issue tracker (dashboard++).

**Comment by nmanovic** [comment]
Ajda Lampe @ajdal Mar 14 22:28
Hello. We've been using tool for a while, but up till now each person had to run docker locally, created their own tasks etc... Now we installed it on server to simplify usage for users who are not comfortable with Linux, but I need help figuring some managing out:

1) How do groups work? Can I set the privileges somehow?
I assumed staff role would enable to manipulate tasks, but not create users. So far it seems like they only see Queues...
2) Is it possible for users to assign tasks to themselves? We have a queue of tasks visible to everyone and as most aren't "full time annotators", it is hard to predict the speed at which they will finish tasks - therefore it would be nice if, when they start working on it, they could assign a task to themselves, so it wouldn't be visible to other users.
3) Does "Status" affect a task in any way? Can a normal user set it? As in set it to "validation" when they finish annotating?

I'm sorry if these are stupid questions, but knowing the answers would simplify my life a lot and I didn't notice it in the docs :)
